### PR TITLE
Tune linters

### PR DESCRIPTION
**Pull request by thabarbados**
### Fix or Enhancement?
- add prettier and tune linting configs
- fix linting errors

- [x] All tests passed

### Environment
- OS: BigSur MacOs
- NPM Version: 6.14.15

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,14 +1,22 @@
 {
```
Comment (dangreen) [inline .babelrc:1]: File is not changed - shouldn't be in diff

File: package.json [hunk 2]
```diff
@@ -59,7 +59,8 @@
     "prepublishOnly": "yarn run lint && yarn run test && yarn run build",
     "docs:dev": "vuepress dev docs",
     "docs:build": "vuepress build docs",
-    "commit": "cz"
+    "commit": "cz",
+    "format": "prettier --write src test"
```
Comment (dangreen) [inline package.json:63]: please move it under `"lint"`

File: package.json [hunk 3]
```diff
@@ -69,7 +70,8 @@
   },
   "devDependencies": {
     "@babel/cli": "^7.0.0-beta.42",
-    "@babel/core": "^7.0.0-beta.42",
+    "@babel/core": "7.5.5",
+    "@babel/plugin-transform-modules-commonjs": "^7.16.0",
```
Comment (dangreen) [inline package.json:74]: for what?

**Review by dangreen — CHANGES_REQUESTED** [review]
Also where is `tests/.eslintrc.js` ?

File: package.json (hunk 2 above)
Comment (thabarbados) [reply package.json]: moved

File: package.json (hunk 3 above)
Comment (thabarbados) [reply package.json]: sorry. it was from previous pr

File: .babelrc (hunk 1 above)
Comment (thabarbados) [reply .babelrc]: rolled back

**Pull request merged**